engine/src/main/java/org/terasology/rendering/shader/ShaderParametersCombine.java

```diff
@@ -107,6 +107,12 @@ public void applyParameters(Material program) {
             program.setFloat("outlineThickness", outlineThickness, true);
         }
 
+        //TODO: Other parameters and volumetric fog test case is needed
+        Camera activeCamera = CoreRegistry.get(WorldRenderer.class).getActiveCamera();
```

Comment:
Thank you. I'd love to change that convention everywhere, it makes for very cluttered and potentially slightly lower performance code. But I do not have the time for a systematic change. I just do it in the code I'm focusing on.